### PR TITLE
SEO-184192 JavaScript Page with Redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have modified title, H1 and description, please review.
Regards,
Yvone.

### CODE REVIEW

File: js/Diagram/Label.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Label | Diagram | Javascript | Syncfusion
-description: This section explains how to use Essential ejDiagram control Label's functionality and its behavior.
+title: Label in Javascript Diagram Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline js/Diagram/Label.md:3]: Update like this - JavaScript

File: js/Diagram/Label.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+description: Learn here all about label support in Syncfusion JavaScript Diagram control, its elements, and more.
 platform: js
 control: Diagram
 documentation: ug
 api: /api/js/ejdiagram
 ---
 
-# Label
+# Label in Javascript Diagram Control
```
Comment (gowthamanthavasiyappan) [inline js/Diagram/Label.md:11]: Update like this - JavaScript

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: js/Diagram/Label.md (hunk 1 above)
Comment (Yvone-Atieno) [reply js/Diagram/Label.md:3]: updated

File: js/Diagram/Label.md (hunk 2 above)
Comment (Yvone-Atieno) [reply js/Diagram/Label.md:11]: updated

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**